woocommerce-pdf-invoices-packingslips.php

```diff
@@ -542,6 +543,34 @@ public function legacy_addon_notices() {
 			}
 		}
 	}
+	
+	public function order_proposal_below_2_0_2_notice() {
+		$transient_name = 'wpo_wcpdf_woocommerce_order_proposal_below_2_0_2_deactivated';
+		$query_arg      = "{$transient_name}_notice";
+
+		if ( get_transient( $transient_name ) ) {			
+			ob_start();
+			?>
+			<div class="notice notice-warning">
+				<p><?php _e( 'Attention! We detected that your WooCommerce Order Proposal plugin is running a version below 2.0.2. In the next version of the PDF Invoices & Packing Slips for WooCommerce plugin, we are making a drastic change that requires at least version 2.0.2 of the Proposals plugin. To avoid clashes, we\'ve temporarily deactivated it—feel free to update to version 2.0.2 or higher. Thank you!', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+				<p><a href="<?php echo esc_url( wp_nonce_url( add_query_arg( array( $query_arg => true ) ), 'hide_order_proposal_below_2_0_2_notice_nonce' ) ); ?>"><?php _e( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ); ?></a></p>
+			</div>
+			<?php
+			echo wp_kses_post( ob_get_clean() );
+		}
```

Comment:
Should it check the Order Proposal plugin version before displaying this notice?